data/campaigns/Descent_Into_Darkness/achievements.cfg

```diff
@@ -1,12 +1,62 @@
 #textdomain wesnoth-did
+
+# achievements in DiD generally should be based around exploration and trying funny things in the campaign
+# these achievements shouldn't be so much about battle challenges, I want to give players another incentive to explore the little details
+# also the achievements are somewhat written as if from Malin's point of view which is why there's some weird and sometimes quirky flavor in them
 [achievement_group]
     display_name=_"Descent into Darkness"
     content_for=descent_into_darkness
     [achievement]
         id = "did_rat_eater"
-        name = _"Rat Eater"
-        description = _"Have a Ghoul eat way too many rats during Descent into Darkness's A Haunting in Winter."
-        icon="data/core/images/icons/potion_red_small.png"
+        name = _"A Big Hunk of Flesh"
+        description = _"Feed a Ghoul way too many rats."
+        icon="data/core/images/attacks/fangs-animal.png"
+        icon_completed="data/core/images/attacks/fangs-animal.png~SCALE(72,72)~BLIT("data/core/images/misc/achievement-frames/frame-8-acid.png",0,0)"
+    [/achievement]
+    [achievement]
+        id = "did_drogan"
+        name = _"Yer a Ghoul, Drogan"
+        description = _"Make that arrogant jerk Drogan into just another one of your minions."
+        icon="data/core/images/attacks/staff-plague.png"
+        icon_completed="data/core/images/attacks/staff-plague.png~SCALE(72,72)~BLIT("data/core/images/misc/achievement-frames/frame-8-acid.png",0,0)"
```

Comment:
I believe `icon=` is unused for hidden achievements so you might as well just remove it. (Or you can set `icon=` to the current value of `icon_completed=` and remove `icon_completed=` – that should give the same effect, but I think it's up to you which seems better.)

This applies to all the hidden achievements.

Reply:
Okay.